src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -76,8 +75,8 @@
   @Inject
   private Logger logger;
   
-  @Inject
-  private OrganizationSettingController organizationSettingController;
+  //@Inject
+  //private transient OrganizationSettingController organizationSettingController;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)